### ISSUE TITLE
Run tests in Travis CI

### ISSUE THREAD

**Issue by Twaha-Rahman** [freeCodeCamp/CamperChan-discord#36]
**Is your feature request related to a problem? Please describe.**

This feature is related to Travis CI[#5] support and tests (with Jest) support[#6] .

**Describe the solution you'd like**

Tests are going to be soon added to the project and it is an important task to connect these tests with Travis CI so it can run tests on on PRs. ~~It would also be a good idea to integrate tests with Git hooks since the foundation is already there.~~

**Additional context**

The idea for integrating Travis and tests were said in #5

**Comment by bradtaniguchi** [comment]
I recommended adding running tests with travis in the add test support using jest PR.

I'm **not a fan** of running a lot of stuff via git hooks, such as testing on commit/push as this slows down an action that should be taken regularly, and often. On some of the larger projects I've worked on, even formatting/linting the code gets turned off as committing takes longer than a few moments. Most of this stuff is checked in the CI/CD service anyways, and should be checked before merging at some point, just maybe not via git hooks.
